ccm_web/client/src/components/ValidationErrorTable.tsx

```diff
@@ -50,7 +54,8 @@ function ErrorTable (props: ErrorTableProps): JSX.Element {
 
     return (
       <TableContainer component={Paper}>
-        <Table stickyHeader aria-label="custom pagination table">
+        <Table stickyHeader aria-label='custom pagination table'>
+          <TableCaption text={`${props.invalidations.length} validation errors occurred.`} />
           <TableHead>
               <TableRow>
                 {columns.map((column) => (
```

Comment:
`ValidationErrorTable` could use `CustomTable`. This was a refactor discussed at one point but did not happen. It should be relatively straightforward if you want me to do it.

Reply:
can the error message be something like `5 validation errors occurred on row 2,3,4,5,9`? 
I am not asking for the change if accessibility team suggested `5 validation errors occurred` is good info for table caption.

Reply:
Sure

Reply:
I'll do this separately.